test/documentSymbols.test.ts

```diff
@@ -275,5 +301,21 @@ describe('Document Symbols Tests', () => {
         createExpectedDocumentSymbol('conditions', SymbolKind.Module, 6, 12, 10, 28, 6, 12, 6, 22, [root2])
       );
     });
+
+    it('Document symbols with a limit', () => {
```

Comment:
Why does this test and the one above on line 141 have the same description?

Reply:
Ah, sorry, I relied on the different `describe` blocks to differentiate them but I see this doesn't match the naming convention in the file. Will fix.

Reply:
OK, thanks. I missed that in the truncated github diff view.